src/components/bridge/ccip/CcipBridge.vue

```diff
@@ -0,0 +1,346 @@
+          <div>
+            <span class="text--to--balance">
+              <token-balance
+                text="assets.modals.balance"
+                :balance="String(toBridgeBalance)"
+                :symbol="selectedToken.symbol"
+              />
+            </span>
+          </div>
+        </div>
+        <div class="box__row">
+          <img width="24" :src="ccipBridgeIcon[toChainName]" alt="chain-icon" />
+          <div class="column--chain">
+            <div>
+              <span class="text--title">{{ toChainName }}</span>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="box--input-field box--hover--active">
+        <div class="box__space-between">
+          <div />
+          <div class="box__available">
+            <span class="text--to--balance">
+              <token-balance
+                text="assets.modals.balance"
+                :balance="String(fromBridgeBalance)"
+                :symbol="selectedToken.symbol"
+              />
+            </span>
+          </div>
+        </div>
+        <div class="box__row">
+          <div class="box__row">
+            <div class="token-logo">
+              <img width="24" alt="token-logo" :src="selectedToken.image" />
+            </div>
+            <span class="text--title">{{ selectedToken.symbol }}</span>
+          </div>
+          <div class="box__column--input-amount">
+            <input
+              :value="bridgeAmt"
+              inputmode="decimal"
+              type="number"
+              min="0"
+              pattern="^[0-9]*(\.)?[0-9]*$"
+              placeholder="0"
+              class="input--amount input--no-spin"
+              @input="(e) => inputHandler(e)"
+            />
+          </div>
+        </div>
+      </div>
+
+      <div v-if="!isApproved && !isNativeToken">
+        <div class="input--checkbox" :class="isApproveMaxAmount && 'input--checkbox--checked'">
+          <input
+            id="approve-max-amount"
+            :checked="isApproveMaxAmount"
+            :value="isApproveMaxAmount"
+            type="checkbox"
+            class="checkbox"
+            @change="(event:any)=>$emit('update:isApproveMaxAmount', event.target.checked)"
+          />
+          <label for="approve-max-amount">
+            <span>{{ $t('bridge.approvalMaxAmount') }}</span>
+          </label>
+        </div>
+      </div>
+
+      <div v-if="errMsg && currentAccount" class="row--box-error">
+        <span class="color--white"> {{ $t(errMsg) }}</span>
+      </div>
+
+      <div class="container--warning">
+        <ul>
+          <li>
+            {{
+              $t('bridge.feeOnBridge', {
+                amount: nativeToken === 'ETH' ? truncate(bridgeFee, 5) : $n(truncate(bridgeFee)),
+                symbol: nativeToken,
+              })
+            }}
+          </li>
+          <li>{{ $t('bridge.warningCcipTime', { time: isToSoneium ? '3' : '30' }) }}</li>
```

Comment:
Use constants for 3 and 30

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1416/commits/e789b2a4b553d16ecea5a1d0499e363fe6fd07b5) commit.